infrastructure/test_config.tf

```diff
@@ -0,0 +1,15 @@
+resource "azurerm_key_vault_secret" "test-scan-delay" {
+  name      = "test-scan-delay"
+  value     = "${var.scan_delay}"
+  vault_uri = "${module.bulk-scan-key-vault.key_vault_uri}"
+
+  depends_on = ["azurerm_storage_account.provider"]
+}
+
+resource "azurerm_key_vault_secret" "test-storage-account-name" {
+  name      = "test-storage-account-name"
+  value     = "${azurerm_storage_account.provider.name}"
+  vault_uri = "${module.bulk-scan-key-vault.key_vault_uri}"
+
+  depends_on = ["azurerm_storage_account.provider"]
+}
```

Comment:
'scan delay' and 'storage account name' are not really secret information.
Consider setting env vars for tests directly.
See https://github.com/hmcts/private-beta-invitation-service/blob/master/infrastructure/output.tf#L13 for reference

Reply:
Thanks, I'll give it a go!